build.assets/Dockerfile-arm

```diff
@@ -1,8 +1,72 @@
-ARG BUILDBOX_VERSION
-FROM public.ecr.aws/gravitational/teleport-buildbox:$BUILDBOX_VERSION
+FROM docker.io/library/debian:11
```

Comment:
> Do we have any documentation around our build process and which artifacts are built from which docker image?

We are working on it. We should have something later this month.

> In what circumstances do we use debian instead of the buildbox and why?

"buildbox" as a name is confusing. We currently have multiple buildboxes. The buildbox that you're referring to uses Ubuntu 22.04 image and should be only used in our CI and not for any releases. We're using Debian 11 here as our binaries must run on distroless which is based on Debian 11.

> Why is the buildbox not just using debian?

Because our CI buildbox should be ONLY used in CI and nowhere else. I want to standardize our Docker images, but in order to do that we need to move out, all builds out of the builbox. These images should not depend on each order, as changes to CI should not break the release (which happened recently).

Reply:
Capturing some of what you said, like the purpose of the image, in a comment here would be nice.